coreneuron/CMakeLists.txt

```diff
@@ -89,10 +89,9 @@ if(CORENRN_ENABLE_GPU)
   # compile cuda files for multiple architecture
   cuda_add_library(
     "cudacoreneuron" ${CORENEURON_CUDA_FILES}
-    OPTIONS -arch=sm_30
-            -gencode=arch=compute_30,code=sm_30
-            -gencode=arch=compute_50,code=sm_50
-            -gencode=arch=compute_52,code=sm_52
+    OPTIONS -arch=sm_60
+            -gencode=arch=compute_60,code=sm_60
+            -gencode=arch=compute_72,code=sm_72
```

Comment:
Nitpick. Maybe we can add `compute_80/sm_80` for `A100` for future-proofing?

Reply:
@iomaganaris : I haven't added that because older CUDA versions then fail to compile. We need to re-work that separately based on CUDA version.